### ISSUE TITLE
Dependency on gbm 2.2 when the latest i can find is 2.1.4.9000

### ISSUE THREAD

**Issue by derek-corcoran-barrios** [patr1ckm/mvtboost#11]
Hello Patrick:

I have been very excited to use your package, but I got into an issue, where I installed but when loading the package I got

Error: package ‘gbm’ 2.1.4.9000 was found, but >= 2.2 is required by ‘mvtboost’

After a long search, it seems that gbm 2.2 is actually the gbm3 package, am I correct, do you think that can be solved?

Cheers

Derek

**Comment by patr1ckm** [comment]
Correct, and it's not going to be an easy solve unfortunately. This was primarily a research prototype that I no longer maintain. Best to move on to explore other approaches.
